lib/saluki-env/src/workload/stores/tag.rs

```diff
@@ -32,40 +34,40 @@ pub struct TagStore {
     snapshot: Arc<ArcSwap<TagSnapshot>>,
 
     entity_limit: NonZeroUsize,
-    active_entities: AHashSet<EntityId>,
+    active_entities: FastHashSet<EntityId>,
```

Comment:
I like the switch to `FastHashSet`. Might be worth documenting why ahash and not some other hash in the prelude.

Reply:
Good idea. Added a blurb to the prelude.